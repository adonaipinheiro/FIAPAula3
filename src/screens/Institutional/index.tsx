import React from 'react';
import {Text, View} from 'react-native';

export default function Institutional() {
  return (
    <View testID="InstitutionalScreen">
      <Text>FIAP</Text>
      <Text>Educação que transforma!</Text>
    </View>
  );
}
