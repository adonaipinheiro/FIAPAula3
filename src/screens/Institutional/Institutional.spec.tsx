import React from 'react';
import {render, screen} from '@testing-library/react-native';

import Institutional from '.';

describe('Institutional', () => {
  beforeEach(() => {
    render(<Institutional />);
  });
  it('should render corretly', () => {
    const institutionalScreen = screen.getByTestId('InstitutionalScreen');
    expect(institutionalScreen).toBeTruthy();
  });
  it('should render title', () => {
    const text = screen.getByText('FIAP');
    expect(text).toBeTruthy();
  });
  it('should Match snapshot', () => {
    const institutionalScreen = screen.toJSON();
    expect(institutionalScreen).toMatchSnapshot();
  });
});
