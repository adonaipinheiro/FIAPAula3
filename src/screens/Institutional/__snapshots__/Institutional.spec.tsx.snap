// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Institutional should Match snapshot 1`] = `
<View
  testID="InstitutionalScreen"
>
  <Text>
    FIAP
  </Text>
  <Text>
    Educação que transforma!
  </Text>
</View>
`;
